src/lib/backend.ai-client-node.js

```diff
@@ -2428,6 +2428,11 @@ class Maintenance {
         this.client = client;
         this.urlPrefix = '/resource';
     }
+    attach_background_task(task_id) {
```

Comment:
@mickowale We need a comment section above this method. (It is referenced by TypeScript to check the parameter type)